lib/dotcom_web/live/trip_planner.ex

```diff
@@ -109,26 +122,75 @@ defmodule DotcomWeb.Live.TripPlanner do
     {:noreply, socket}
   end
 
-  # Selected from list of popular locations
-  defp location_props(%{"stop_id" => stop} = props) when is_binary(stop) do
-    Map.take(props, ["name", "latitude", "longitude", "stop_id"])
+  defp update_from_pin(socket, %{
+         "from" => %{"longitude" => from_longitude, "latitude" => from_latitude}
+       }) do
+    update_pin_in_socket(socket, [from_longitude, from_latitude], 0)
```

Comment:
Why change lat/long from a named map to an array? 

My concern is that while geojson does have the standard of `[long, lat]`, `[lat, long]` is a lot more common in colloquial usage. (See Google Maps screenshot below, for instance). Because of that, just a list of numbers risks a very easy-to-make and hard-to-protect-against error of accidentally typing in `[lat, long]` instead of `[long, lat]`.

<img width="207" alt="Screenshot 2024-11-01 at 11 45 15 AM" src="https://github.com/user-attachments/assets/27c1319d-c3d4-4959-9960-5bffd064b081">

Unless there's a good reason to follow the geojson standard, I'd strongly prefer to keep the named map all the way through.

Reply:
Maplibregl uses lng, lat. Since we're using that library, it makes sense to use that too. See the definition of `center` in the map config.

Reply:
Ah yeah - my comment wasn't clear enough. I saw that MapLibre (which we don't control) and Metro (which we do) use geojson format. What I'd prefer is to move the geojson usage to be closer to the underlying library call, rather than leaking it all the way through the live component.

I made a suggested refactor that does address this, but if you don't like that refactor, then let's revisit this conversation.